CHANGELOG_OLD.md

```diff
@@ -0,0 +1,1451 @@
+## unreleased
```

Comment:
This is just a copy of the deleted `CHANGELOG.md` right? I was initially confused by the fact that they have a different number of lines and have different content. I'd imagine a re-base should clear it up.

Reply:
oh yeah, this was meant to just be a `mv` of the old changelog, the difference probably popped up after I rebased. I'll fix.